src/ConfigConverter/Commands/ConvertConfigToAsciiDocFormatCommand.php

```diff
@@ -43,7 +43,8 @@
  */
 class ConvertConfigToAsciiDocFormatCommand extends Command
 {
-    const FILTER_REGEX = '/(\$CONFIG = \[)([\S\s|\n|\r]*)(\];)/';
+    const FILTER_REGEX_1 = '/(\$CONFIG = \[)([\S\s|\n|\r]*)(\];)/';			# array is defined with [ ]
```

Comment:
I'm not keen on magical suffixes such as `_1` and `_2`. Pls change the names to be more meaningful/intuitive.

Reply:
was thinking about too but how should I name them ?
I prepared the other constant because when core changes from array() to [] it is easy to fix here.